scenes/tileset.tres

```diff
@@ -29,12 +29,17 @@ tile_data = PackedInt32Array(0, 327681, 3, 1, 393217, 3, 2, 458753, 3)
 [sub_resource type="TileSetAtlasSource" id="TileSetAtlasSource_sugp2"]
 texture = ExtResource("1_glnwb")
 texture_region_size = Vector2i(64, 64)
-0:0/next_alternative_id = 2
+0:0/next_alternative_id = 3
 0:0/0 = 0
 0:0/0/z_index = -1
 0:0/0/physics_layer_0/polygon_0/points = PackedVector2Array(-32, -32, 32, -32, 32, 32, -32, 32)
+0:0/0/physics_layer_1/polygon_0/points = PackedVector2Array(-32, -32, 32, -32, 32, 32, -32, 32)
 0:0/1 = 1
 0:0/1/z_index = -1
+0:0/1/physics_layer_0/polygon_0/points = PackedVector2Array(-32, -32, 32, -32, 32, 32, -32, 32)
+0:0/1/physics_layer_1/polygon_0/points = PackedVector2Array(-32, -32, 32, -32, 32, 32, -32, 32)
+0:0/2 = 2
+0:0/2/physics_layer_0/polygon_0/points = PackedVector2Array(-32, 3.85675, 32, 4.40771, 32, 32, -32, 32)
```

Comment:
I understand that the purpose of adding the second alternative is to have a "half-height" collision shape for the water at the bottom of the bridge.

However I think that your edit to the first alternative is wrong. I am pretty sure that the whole point of the first water alternative tile is that it has no collision shape, so it can be placed under bridges. I have checked and this tile is the one that is under the bridges in the stealth quest, and sure enough they aren't walkable after this change.

Reply:
In-person conversation confirmed that this was a mistake, and I've fixed this.